src/lib/y2storage/resize_info.rb

```diff
@@ -32,8 +33,33 @@ module Y2Storage
   # This is a wrapper for Storage::ResizeInfo
   class ResizeInfo
     include StorageClassWrapper
+    include Yast::I18n
+    extend Yast::I18n
     wrap_class Storage::ResizeInfo
 
+    # rubocop:disable Metrics/LineLength
+    REASON_TEXTS =
+      {
+        RB_RESIZE_NOT_SUPPORTED_BY_DEVICE:       N_("Resizing is not supported by this device."),
+        RB_MIN_MAX_ERROR:                        N_("Combined limitations of partition and filesystem prevent resizing."),
+        RB_SHRINK_NOT_SUPPORTED_BY_FILESYSTEM:   N_("This filesystem does not support shrinking."),
+        RB_GROW_NOT_SUPPORTED_BY_FILESYSTEM:     N_("This filesystem does not support growing."),
+        RB_FILESYSTEM_INCONSISTENT:              N_("Filesystem consistency check failed."),
+        RB_MIN_SIZE_FOR_FILESYSTEM:              N_("This filesystem already has the minimum possible size."),
+        RB_MAX_SIZE_FOR_FILESYSTEM:              N_("This filesystem already has the maximum possible size."),
+        RB_FILESYSTEM_FULL:                      N_("The filesystem is full."),
+        RB_NO_SPACE_BEHIND_PARTITION:            N_("There is no space behind this partition."),
+        RB_MIN_SIZE_FOR_PARTITION:               N_("This partition already has the minimum possible size."),
+        RB_EXTENDED_PARTITION:                   N_("Extended partitions cannot be resized."),
+        RB_ON_IMPLICIT_PARTITION_TABLE:          N_("The partition on an implicit partition table cannot be resized."),
+        RB_SHRINK_NOT_SUPPORTED_FOR_LVM_LV_TYPE: N_("Shrinking of this type of LVM logical volumes is not supported."),
+        RB_RESIZE_NOT_SUPPORTED_FOR_LVM_LV_TYPE: N_("Resizing of this type of LVM logical volumes is not supported."),
+        RB_NO_SPACE_IN_LVM_VG:                   N_("No space left in the LVM volume group."),
```

Comment:
I expect this is for increase of size. So maybe hint them that they need to shrink another logical volume before increasing size of this one?

Reply:
Or add another physical volume. Or delete a logical volume. We give reasons why the operation the user requested doesn't work; he will have to figure out what to do about it.

This is an error message, not a troubleshooting guide or a generic manual about storage. ;-)

Reply:
I am not saying to give always bullet proof manual, but at least some hint would be nice. It is always better to get "We cannot do this because ABC. Doing DEF can help." than "We cannot do this because ABC". But it is not blocker for me, it is mainly Nice to Have.